src/views/AverageWaveforms/AverageWaveformsView.tsx

```diff
@@ -17,18 +18,20 @@ const AverageWaveformsView: FunctionComponent<Props> = ({data, width, height}) =
     const setSelectedPlotKeys = useCallback((keys: string[]) => {
         setSelectedUnitIds(keys.map(k => (Number(k))))
     }, [setSelectedUnitIds])
-    const plots = useMemo(() => (data.averageWaveforms.map(aw => ({
+    const plots = useMemo(() => (data.averageWaveforms.sort((a1, a2) => (a1.unitId - a2.unitId)).map(aw => ({
         key: `${aw.unitId}`,
         label: `Unit ${aw.unitId}`,
+        labelColor: colorForUnitId(aw.unitId),
         props: {
             channelIds: aw.channelIds,
-            waveform: aw.waveform,
+            waveform: subtractChannelMeans(aw.waveform),
```

Comment:
Yes, the reason to do this... if the raw traces is not bandpass filtered, then the avg waveform can have an offset far from zero, and for display purposes we'd like to subtract off the mean for each channel. It might not be the best decision, but it was needed for certain datasets I was working with. We can revisit down the road.